icd/settings/settings_xgl.json

```diff
@@ -1879,6 +1879,21 @@
       "VariableName": "usePalPipelineCaching",
       "Scope": "Driver"
     },
+    {
+      "Name": "EnableInternalPipelineCachingToDisk",
+      "Description": "Controls whether the pipeline compiler enables Pal's archive-file based caching for internal pipelines. (Default: TRUE) Related environment variables AMD_ARCHIVE_DISK_CACHE_PATH: Path to where archive file is to be stored (optional) AMD_ARCHIVE_APP_PREFIX     : Fixed prefix string for generated archive file name (optional)",
+      "Tags": [
+        "SPIRV Options"
+      ],
+      "Defaults": {
+        "Default": true,
+        "WinDefault": true,
+        "LnxDefault": true
```

Comment:
You don't need OS specific defaults since they are the same.  I'm sure your copying AllowVkPipelineCachingToDisk, and I've asked for that to be cleaned up.